flight/PiOS/Common/pios_px4flow.c

```diff
@@ -0,0 +1,286 @@
+/**
+ ******************************************************************************
+ * @addtogroup PIOS PIOS Core hardware abstraction layer
+ * @{
+ * @addtogroup PIOS_PX4FLOW PX4Flow Functions
+ * @brief Deals with the hardware interface to the PixHawk optical flow sensor
+ * @{
+ * @file       pios_px4flow.c
+ * @author     Kenn Sebesta, Copyright (C) 2014
+ * @brief      PX4Flow optical flow sensor
+ *
+ ******************************************************************************
+ */
+
```

Comment:
Should have an appropriate license header, e.g. these lines

```
 * @see        The GNU Public License (GPL) Version 3
 *
 *****************************************************************************/
/*
 * This program is free software; you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation; either version 3 of the License, or
 * (at your option) any later version.
 *
 * This program is distributed in the hope that it will be useful, but
 * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
 * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
 * for more details.
 *
 * You should have received a copy of the GNU General Public License along
 * with this program; if not, write to the Free Software Foundation, Inc.,
 * 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 */
```